docs/governance/mrpowers-io-org.md

```diff
@@ -0,0 +1,40 @@
+# mrpowers-io GitHub organization
+
+The mrpowers-io GitHub organization is owned by [Matthew Powers](https://www.linkedin.com/in/matthew-powers-cfa/) and [Sem Sinchenko](https://www.linkedin.com/in/semyon-a-sinchenko/) ("org owners").
```

Comment:
I would like to have here a link to my github account, not Linkedin. The reason is that not all of potential contributors may have linkedin account but all of the should have a git account.

Reply:
Updated.